ga4gh/cli.py

```diff
@@ -401,10 +434,10 @@ def _textOutput(self, gaObjects):
             # TODO add in some more useful output here.
             print(read.id)
 
-
 # ListReferenceBases is an oddball, and doesn't fit either get or
 # search patterns.
 
+
```

Comment:
hm, interesting... I wonder why this wasn't failing before, then...

make sure you're using the same version of flake that is specified in `requirements.txt`, if you aren't already...